catroid/build.gradle

```diff
@@ -95,9 +95,6 @@ ext.useAndroidLocales = { ->
     rootProject.hasProperty('useAndroidLocales') ? 'true' : 'false'
 }
 
-ext.useWebTest = { ->
-    rootProject.hasProperty('useWebTest') ? 'true' : 'false'
-}
 
```

Comment:
One empty line has to be deleted see https://jenkins.catrob.at/job/Catroid/job/PR-3433/2/analysis/fileName.1954838425/source.edbe3ca6-d672-4517-8ceb-05e208e36735/#96